lib/engine/game/g_18_va/game.rb

```diff
@@ -0,0 +1,708 @@
+# frozen_string_literal: true
+
+require_relative 'meta'
+require_relative '../base'
+
+if RUBY_ENGINE == 'opal'
+  require_tree 'engine/game/g_18_va/step'
```

Comment:
why do you need to require this stuff? do other games do this?

Reply:
Only 18FL (which I cloned 18VA from). I removed it from both and it worked. I also removed the Assign step (for now) since that's not needed to make the build work anymore